includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -118,6 +118,28 @@ private static function get_donation_order_item( $frequency, $amount = 0 ) {
 		return $item;
 	}
 
+	/**
+	 * Get WC's Order Item related to a given SKU.
+	 * 
+	 * @param string $product_sku Product's SKU string.
+	 * @param number $amount Donation amount.
+	 * 
+	 * @return boolean|WC_Order_Item_Product Order item product, or false if there's no product with matching SKU.
+	 */
+	public static function get_order_item_by_sku( $product_sku, $amount = 0 ) {
```

Comment:
1. A method should be private by default, since it's not used outside of this class. This helps in future refactoring.
2. `get_order_item_by_sku` and `get_donation_order_item` are almost identical – could be refactored to a function that takes a product ID and amount and creates the item.

Reply:
Good call! The method is public because I have a migration script (not in this repo) that needs to call it directly. But I'll look into refactoring these two methods into a single, more abstract one.